typescript/codepipeline-build-deploy/lib/codepipeline-build-deploy-stack.ts

```diff
@@ -205,14 +226,26 @@ export class CodepipelineBuildDeployStack extends cdk.Stack {
       stageName: "Source",
       actions: [
         new pipelineactions.CodeCommitSourceAction({
-          actionName: "CodeCommit",
+          actionName: "AppCodeCommit",
           branch: "main",
-          output: sourceArtifact,
+          output: SourceArtifact,
```

Comment:
Looks like you meant this to be lowercase
```suggestion
          output: sourceArtifact,
```